### DISCUSSION TITLE
How can I run lammps with deepmd-kit

### DISCUSSION THREAD

**Discussion by guesstimate028** [deepmodeling/deepmd-kit#439] [Q&A]
I install deepmd-kit 1.3.2 with conda and freeze a modle in graph.pb. Then I want to run lammps using "pair_style deepmd graph.pb" in in.SI. It says "ERROR: Unrecognized pair style 'graph.pb'". How can I solve these? plz help

**Comment by njzjz** [comment]
It's not an expected behavior. Could you post more information?

**Reply by guesstimate028** [reply]
Is it necessary for running lammps that deepmd-kit is install with C++? And I can't install it following the tips. When I print "cmake -DTENSORFLOW_ROOT=$tensorflow_root -DCMAKE_INSTALL_PREFIX=$deepmd_root ..", it says "CMake Error at cmake/Findtensorflow.cmake:13 (string):". How can I solve it?

**Reply by AnguseZhang** [reply]
If you use conda installation package to install DeePMD-kit, there's no need to compile DeePMD-kit's C++ interface. Because a proper patched version of executable lammps has already installed via the installation package. 
Note that you should use THIS VERSION of patched lammps , instead of your own lammps or other other version of DeePMD-kit + lammps.

**Reply by AnguseZhang** [reply]
If you still have problem, you are suggested provide the version of DeePMD-kit (how you installed) and necessary scripts & input files to reproduce your problem.

**Reply by guesstimate028** [reply]
I have installed another lammps. How can I use the version of patched lammps？My version of deepmd-kit is 1.2.2.

**Reply by njzjz** [reply]
> I have installed another lammps. How can I use the version of patched lammps？My version of deepmd-kit is 1.2.2.

Execuate `<install_path>/bin/lmp -h` and you will see the lammps help.
